spec/puppetserver/ca/config/puppet_spec.rb

```diff
@@ -338,4 +338,41 @@
       expect(conf.default_certname).to eq("foo")
     end
   end
+
+  it 'loads masterport correctly' do
+    masterport = 1234
+    Dir.mktmpdir do |tmpdir|
+      puppet_conf = File.join(tmpdir, 'puppet.conf')
+      File.open puppet_conf, 'w' do |f|
+        f.puts(<<-INI)
+          [server]
+            masterport = #{masterport}
```

Comment:
(nit) Does it make sense to put this setting in the `master` section instead of `server`? I'm imagining that someone would probably either make all the server->master changes or none of them? But it probably doesn't really matter here, especially if we're already testing the section renames elsewhere.

Reply:
I don't think it matters, particularly because we just rename [master] to [server] internally when needed before any other processing happens.